index.bs

```diff
@@ -128,7 +123,7 @@ spec: webidl
       or other sources this specification hasn't anticipated.
     </dd>
 
-    <dt><dfn export>Powerful feature</dfn></dt>
+    <dt><dfn export lt="powerful feature|feature">Powerful feature</dfn></dt>
```

Comment:
Yes, `<dfn export local-lt=feature>Powerful feature</dfn>` is the ideal markup here.